.github/workflows/test-env-deploy.yml

```
@@ -251,15 +253,27 @@ jobs:
           CYPRESS_SECOND_USER_NAME: ${{ secrets.CYPRESS_SECOND_USER_NAME }}
           CYPRESS_USER_PASSWORD: ${{ secrets.CYPRESS_USER_PASSWORD }}
           CYPRESS_PERMISSIONS_USERS_PASSWORD: ${{ secrets.CYPRESS_PERMISSIONS_USERS_PASSWORD }}
-          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           CYPRESS_MAILPITURL: ${{ secrets.CYPRESS_MAILPITURL }}
-          COMMIT_INFO_MESSAGE: ${{github.event.inputs.tags}} tests triggered on PR - https://github.com/${{ github.repository }}/pull/${{ github.ref_name }}
           CYPRESS_grepTags: ${{ needs.prepare-tests.outputs.tags }}
+          SPLIT: ${{ strategy.job-total }}
+          SPLIT_INDEX: ${{ strategy.job-index }}
+      - name: Create reports dir
+        if: ${{ success() || failure() }}
```

Docs suggest using `!cancelled()` instead which is also much shorter: https://docs.github.com/en/actions/learn-github-actions/expressions#always

```suggestion
        if: ${{ !cancelled() }}
```